gui/wxpython/core/debug.py

```diff
@@ -61,7 +61,7 @@ def msg(self, level, message, *args):
         :param args: formatting params
         """
         # self.SetLevel()
-        if self.debuglevel > 0 and level > 0 and level <= self.debuglevel:
+        if self.debuglevel > 0 and 0 < level <= self.debuglevel:
```

Comment:
Isn't this sufficient:
```suggestion
        if 0 < level <= self.debuglevel:
```
It should only be true if self.debuglevel > 0, no?

Reply:
You're right. I knew there were an extra simplification to do at some point, I found the pattern weird, so just it in case I misunderstood. Written like that, I can't see where it wouldn't be equivalent.